doc/source/operations/rabbitmq.rst

```diff
@@ -2,6 +2,8 @@
 RabbitMQ
 ========
 
+FIXME
```

Comment:
As Alex has now documented the rabbit migrations here: https://github.com/stackhpc/stackhpc-kayobe-config/pull/1250 I think we can just delete these docs. Especially as quorum queues are handled by a script now

Reply:
Removed